index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Porfolio Website</title>
+    <title>Portfolio Website</title>
```

Comment:
- [ ] First project is overlapping the navigation when I scroll down. When I hover over each project they are also overlapping the navigation. Please fix issue by changing z index ✌🏼

Screenshot
![image](https://user-images.githubusercontent.com/69011963/172342867-9b1d6d6d-7164-4781-90b3-2904844a6f5a.png)

Reply:
@Rustamxon7 Thank you for reviewing my project, I will make the necessary corrections as suggested